website/src/index.html

```diff
@@ -0,0 +1,210 @@
+                        Time to Leave
+                        <img src="images/resources/timer.svg" alt="Time To Leave Logo">
+                    </h1>
+
+                    <p class="marketing__desc">
+                        Time to Leave is a Desktop app for work time management, looking similar to a time-card system.
+                        It’s
+                        different from other software available as it’s very simple, doesn’t require any database
+                        configuration, and focuses on allowing you to control your time. Its premise is to help you
+                        manage
+                        the time you are working and to notify you when it’s time to leave (therefore the name).
+                    </p>
+
+                    <div class="intro__buttons">
+                        <a href="https://github.com/thamara/time-to-leave" class="button--primary">Source code</a>
+
+
+                        <div class="dropdown">
+                            <button id="js-dropdown" class="button--dropdown">Download</button>
+                            <ul>
+                                <li><a href="#">Mac OS X 10.11+</a></li>
+                                <li><a href="#">Windows 7+</a></li>
+                                <li><a href="#">Linux Debian</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="intro__illustration">
+                <img src="./images/HeroTimetoLeave.png" alt="">
+            </div>
+        </section>
+
+
+        <section class="customization">
+            <div class="marketing js-customization--trigger">
+                <div class="marketing__container">
+                    <h2 class="marketing__title js-reveal">Fully customizable</h2>
+                    <p class="marketing__desc js-reveal">
+                        It is highly customizable, not only on the look and feel (with different month and day visions,
+                        different themes, and more), but also offers lots of functionalities such as defining how many
+                        hours
+                        you
+                        usually work per day, which week days, allow you to quickly source holidays based on your region
+                        (Country, State, and City), and much more.
+                    </p>
+
+                    <div class="marketing__features js-reveal">
+                        <div class="marketing__feature">
+                            <img class="marketing__feature__img" src="images/theme.svg" alt="Time To Leave Logo">
+                            <h3 class="marketing__feature__title">Themes</h3>
+                            <p class="marketing__feature__desc">Time to Leave comes with three themes right out the box.
+                                Light, Dark, and are new Cadent
+                                Start theme. </p>
```

Comment:
```suggestion
                                Light, Dark, and Cadent Star themes. </p>
```

Not sure if we meant to say "our new" or something else, but phrasing is weird here, so I just proposed to remove it.